tests/integration.rs

```diff
@@ -53,15 +53,15 @@ fn proper_initialization() {
     let mut deps = mock_instance(WASM);
 
     let msg = InitMsg { count: 17 };
-    let params = mock_params(&deps.api, "creator", &coin("1000", "earth"), &[]);
+    let env = mock_env(&deps.api, "creator", &coin("1000", "earth"), &[]);
 
     // we can just call .unwrap() to assert this was a success
-    let res = init(&mut deps, params, msg).unwrap();
+    let res = init(&mut deps, env, msg).unwrap();
     assert_eq!(0, res.messages.len());
 
     // it worked, let's query the state
     let res = query(&mut deps, QueryMsg::GetCount {}).unwrap();
-    let value: CountResponse = from_slice(&res).unwrap();
+    let value: CountResponse = from_slice(&res.as_slice()).unwrap();
```

Comment:
This now creates a reference to a reference (`&&[u8]`). I don't know if/why that works. But `from_slice(res.as_slice()).unwrap();` is sufficient and simpler